operator/api/v1beta1/lokistack_types.go

```diff
@@ -343,9 +343,30 @@ type ObjectStorageSecretSpec struct {
 	Name string `json:"name"`
 }
 
+// ObjectStorageSchemaVersion defines the storage schema version which will be used with the Loki cluster.
+//
+// +kubebuilder:validation:Enum=v11;v12
+type ObjectStorageSchemaVersion string
+
+const (
+	// ObjectStorageSchemaV11 when using v11 for the storage schema
+	ObjectStorageSchemaV11 ObjectStorageSchemaVersion = "v11"
+
+	// ObjectStorageSchemaV12 when using v12 for the storage schema
+	ObjectStorageSchemaV12 ObjectStorageSchemaVersion = "v12"
+)
+
 // ObjectStorageSpec defines the requirements to access the object
 // storage bucket to persist logs by the ingester component.
 type ObjectStorageSpec struct {
+	// Type of object storage that should be used
+	//
+	// +required
+	// +kubebuilder:validation:Required
+	// +kubebuilder:default:=v12
```

Comment:
I wonder why the default is `v12`, while the example stacks in hack have still `schemaVersion: v11`.

Reply:
Good catch. I was testing the appending schema functionality and forgot to change it back.